action-whitelist/README.md

```diff
@@ -0,0 +1,70 @@
+# action-whitelist
```

Comment:
```suggestion
# action-whitelist

```

Just a preference - whitespace after a heading. Also around code blocks such as after line15. If using VS Code, I recommend extension `davidanson.vscode-markdownlint`.

I also prefer wrapping text around 80 characters or so - that way when reading the plain text, there is less need for the eyes to travel horizontally across the page. For that I use extension `stkb.rewrap`.